src/gui/README.md

```diff
@@ -521,15 +545,23 @@ Statuses:
     200: successful spend. NOTE: the response may include an "error" field. if this occurs, the spend succeeded
          but the response data could not be prepared. The client should NOT spend again.
     400: Invalid query params, wallet lacks enough coin hours, insufficient balance
+    403: Wallet api disabled
     404: wallet does not exist
     500: other errors
 ```
 
 example, send 1 coin to `2iVtHS5ye99Km5PonsB42No3pQRGEURmxyc` from wallet `2017_05_09_ea42.wlt`:
 
 ```sh
-curl -X POST \
-  'http://127.0.0.1:6420/wallet/spend?id=2017_05_09_ea42.wlt&dst=2iVtHS5ye99Km5PonsB42No3pQRGEURmxyc&coins=1000000'
+# get csrf first
```

Comment:
csrf is optional and only matters for the browser interface.  the csrf section should describe how to obtain a csrf token.  remove the csrf doc from all of these endpoints.

in the integration.md doc, add a note to -disable-csrf